app/views/add-reduction.html

```diff
@@ -0,0 +1,156 @@
+{% extends "includes/layout.html" %} {% block content %}
```

Comment:
Minor: Formatting. Newline between includes and block start?

Reply:
👍